Cargo.toml

```diff
@@ -25,10 +25,11 @@ tokio = "0.1"
 websocket = "0.22"
 
 [dependencies.cursive]
-version = "0.12.1-alpha.0"
+version = "0.13.0"
 default-features = false
 features = ["crossterm-backend"]
 git = "https://github.com/gyscos/Cursive.git"
+tag = "v0.13.0"
```

Comment:
If we're limiting it to this tag, how about just removing the optional git version altogether? I think the only reason it was here was to avoid bugs in `0.12.1-alpha.0` that were fixed but not released.

I'd keep the `screeps-api` git version as that crate is developed concurrently with this one, but this one should go.

Reply:
I was trying to bring it back to the state where it definitely built. I'm not sure if this limitation is required actually. It seems like the git source for cargo just compares the version in HEAD's Cargo.toml to the requested version and fails if it doesn't match, which was causing a build failure. Since I wanted to break one thing at a time I just put the tag here.